cmd/networking/ns/main.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"knative.dev/serving/pkg/reconciler/namespace"
```

Comment:
We should have a more specific/descriptive location for this, e.g. `ns-cert` or `nscert`.

Reply:
Whatever name we choose for this should be echoed in the directory this lives within.

Reply:
I went with nscert and changed the directory name.